meson.build

```diff
@@ -506,14 +506,15 @@ if get_option('enable-test')
   )
 
   path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_join_plugin = join_paths(meson.build_root(), 'gst', 'join')
```

Comment:
I recommend the below style.
```
diff --git a/meson.build b/meson.build
index 8ee8623..10e7ced 100644
--- a/meson.build
+++ b/meson.build
@@ -505,7 +505,9 @@ if get_option('enable-test')
     configuration: nnstreamer_test_conf
   )
 
-  path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_plugin = join_paths(meson.build_root(), 'gst')
+  path_ext_plugin = join_paths(meson.build_root(), 'ext')
+
   path_nns_conf = join_paths(meson.build_root(), 'nnstreamer-test.ini')
   path_nns_plugin_prefix = join_paths(meson.build_root(), 'ext', 'nnstreamer')
   path_nns_plugin_filters = join_paths(path_nns_plugin_prefix, 'tensor_filter')
@@ -513,7 +515,7 @@ if get_option('enable-test')
   path_nns_plugin_converters = join_paths(path_nns_plugin_prefix, 'tensor_converter')
 
   testenv = environment()
-  testenv.set('GST_PLUGIN_PATH', path_gst_plugin + ':' + path_nns_plugin_prefix)
+  testenv.set('GST_PLUGIN_PATH', path_gst_plugin + ':' + path_ext_plugin)
   testenv.set('NNSTREAMER_CONF', path_nns_conf)
   testenv.set('NNSTREAMER_FILTERS', path_nns_plugin_filters)
   testenv.set('NNSTREAMER_DECODERS', path_nns_plugin_decoders)
```

You may refer to `debian/rules` and `packaging/nnstreamer.spec` to set `GST_PLUGIN_PATH`.

Reply:
I agree DJ's suggestion

Reply:
I updated the code to follow your kind suggestion. THANKS!